.github/merge_rules.json

```diff
@@ -99,6 +99,16 @@
       "approved_by": ["nikitaved", "cpuhrsch", "pearu", "IvanYashchuk"],
       "mandatory_checks_name": ["Facebook CLA Check", "Lint"]
    },
+   {
+      "name": "Lazy Tensor",
+      "patterns": [
+         "torch/csrc/lazy/**",
+         "test/cpp/lazy/**",
+         "/test/lazy/**"
```

Comment:
is `tools/codegen/*` already covered by some other rule?  if not, we could cover just our specific files, i guess (as there is no actual 'folder' pattern that describes the lazy codegen separately from the other codegens)

Reply:
right! should I add these 4 or is there anything else? 

```
codegen/api/lazy.py
codegen/dest/lazy_ir.py
codegen/dest/lazy_ts_lowering.py
codegen/gen_lazy_tensor.py
```

Reply:
i think that is all for codegen, but also ts_native_fucntions.yaml (aten/native/ i think)